src/components/WwwFrame/TheHeader.vue

```diff
@@ -472,7 +472,7 @@ export default {
 	},
 	watch: {
 		isVisitor(newVal, oldVal) {
-			if (newVal !== oldVal && !newVal) {
+			if (newVal !== oldVal && !newVal && this.$refs.userDropdown) {
```

Comment:
This was throwing a console error. on the GetStarted results page since the ref is missing